e2e/tests/functional/planning/timelist.e2e.spec.js

```diff
@@ -21,7 +21,19 @@
  *****************************************************************************/
 
 import { createDomainObjectWithDefaults, createPlanFromJSON } from '../../../appActions.js';
+import { getEarliestStartTime } from '../../../helper/planningUtils';
 import { expect, test } from '../../../pluginFixtures.js';
+import examplePlanSmall3 from '../../../test-data/examplePlans/ExamplePlan_Small3.json';
```

Comment:
Need to update this to be ESM style and path agnostic (as we do everywhere else)
```suggestion
const examplePlanSmall = JSON.parse(
  fs.readFileSync(new URL('../../test-data/examplePlans/ExamplePlan_Small3.json', import.meta.url))
);
```